### ISSUE TITLE
Cropper view is not displayed properly on iPad splitview

### ISSUE THREAD

**Issue by f0enix** [ZhongUncle/SwiftUI_ImageCropper#2]
when trying the demo app in split view mode on iPad simulator, the cropper view is not properly rendered. The image and cropper overlay is out of the screen. I think the cropper view should use a geometry reader to figure out inside how much space it is being rendered and its position to be adjusted properly.
![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-11-07 at 10 15 18](https://user-images.githubusercontent.com/72214428/200317185-d4fc1f10-c64e-418d-bf2c-d9ed16ec76d4.png)

**Comment by ZhongUncle** [comment]
The aim of size of screen is not only for crop UI, but also for crop scale of image. The ratio of height and width in point is wrong now, I have reported to Apple.
The main reason of using the size of screen is the value is constant value, and constant will use fewer performance than variants.
Yes, at first I try to use the size of background(it will adapt the display automatically). If using the value, after every time we change the View(such as drag and scale), the data flow of structure will calculate and refresh entirely, rather than just one value.
In the past, I made some experiments in my other project. If we use more than just 20 simple Views with simple action at the same time(even many view is invisible in ZStack), almost all iPhone work in low frame(about 10fps) when I drag a view, even the entire app broken down. 
And the project just is a demo, it will add some other functions. I must leave enough performance for other view and function in advance, so I need to consider the details. But you remind me iPad have larger memory and faster CPU and GPU, so I can just use the size of background on iPad.
